src/tests/test_day2.py

```diff
@@ -11,6 +11,8 @@ class TestDay2(BaseTest):
     @JunitTestSuite()
     def test_deploy_day2_nodes_cloud(self, cluster, day2_cluster, controller):
         if not global_variables.cluster_id:
+            cluster.nodes.destroy_all_nodes()
```

Comment:
@eliorerz why we need the destroy here?

Reply:
Because this flow isn't using the global resources.  It will be fixed as part of the day2 pr